examples/rustls-mtls/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "rustls-mtls"
+version = "0.1.0"
+edition = "2021"
+authors = ["Rick Richardson <rick.richardson@gmail.com>"]
```

Since we will maintain this long term and we dont want to confuse others, can you also add our team to the list.

```suggestion
authors = ["Rick Richardson <rick.richardson@gmail.com>", "AWS s2n"]
```

---

Done.